components/gitpod-db/go/cost_center.go

```diff
@@ -233,6 +218,19 @@ func (c *CostCenterManager) UpdateCostCenter(ctx context.Context, newCC CostCent
 	return newCC, nil
 }
 
+func (c *CostCenterManager) getPreviousSpendingLimit(attributionID AttributionID) (int32, error) {
+	var previousCostCenter CostCenter
+	// find the youngest cost center with strategy='other'
+	db := c.conn.Where("id = ? AND billingStrategy = ?", attributionID, CostCenter_Other).Order("creationTime DESC").Limit(1).Find(&previousCostCenter)
+	if db.Error != nil {
```

Comment:
```suggestion
	db := c.conn.
		Where("id = ? AND billingStrategy = ?", attributionID, CostCenter_Other).
		Order("creationTime DESC").
		Limit(1).
		Find(&previousCostCenter)
	if db.Error != nil {
```
would make this more readable